src/ImageTransformations.jl

```diff
@@ -1,3 +1,40 @@
+"""
+
+This package provides support for image resizing, image rotation, and other spatial transformations of arrays.
+
+- `InvWarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[inv(tinv)(I)]`.
+- `WarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[tform(I)]`.
+- `imresize`: Upsample/downsample the image `img` to a given size `sz` or axes `inds` using interpolations.
+- `restrict`: A much more efficient version of `imresize` that two-folds/down-samples image to approximate 1/2 size. (This is now provided by ImageBase.)
```

Comment:
Maybe we should take this one out now, since it has recently migrated to ImageBase.

Reply:
I think we still need this one since it's highly related to this package. So that people can know there's a better version of `imresize` without hanging around in tens of JuliaImages packages.

Reply:
Good point.

Reply:
Keeping it. Edited end of docstring to 'ImageBase.restrict' for ease of search. Thanks for the input!